kernel-crawler/rhel-subscription-crawler/crawler.py

```diff
@@ -226,9 +232,10 @@ def filter_repos(self, content_sets):
         return repos
 
 
-    def get_packages(self, repos):
+    def get_packages(self, repos, session):
         urls = set()
         for repo in repos:
+            logger.info(f'Processing repo {repo}')
```

Comment:
Maybe I remember this incorrectly but I don't think we want any log or print statements other than the package urls. Any print statement will be mistaken for a package url by other steps.